process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -395,7 +395,8 @@ func (st *storageBootstrapper) applyBootInfos(bootInfos []bootstrapStorage.Boots
 		st.forkDetector.SetFinalToLastCheckpoint()
 	}
 
-	err = st.nodesCoordinator.LoadState(bootInfos[0].NodesCoordinatorConfigKey, bootInfos[0].LastHeader.GetEpoch())
+	lastEpoch := getBootInfosLastEpoch(bootInfos)
+	err = st.nodesCoordinator.LoadState(bootInfos[0].NodesCoordinatorConfigKey, lastEpoch)
```

Comment:
can you rename getBootInfosLastEpoch into something like `initNodesCoordinator` and move the call
err = st.nodesCoordinator.LoadState(bootInfos[0].NodesCoordinatorConfigKey, lastEpoch)
inside after checking len(bootInfos) ?

Reply:
remove that function since it was not relevant to have that check, since here we get only if we have at least some entry in boot infos